doc/build-windows-mingw.md

```diff
@@ -0,0 +1,246 @@
+Thread model: posix
+gcc version 4.9.2 (i686-posix-dwarf-rev1, Built by MinGW-W64 project)
+
+
+
+2. Download, unpack and build required dependencies.
+Save them in c:\deps folder.
+
+2.1 OpenSSL: http://www.openssl.org/source/openssl-1.0.1k.tar.gz
+From a MinGw shell (C:\MinGW\msys\1.0\msys.bat), unpack the source archive with tar (this will avoid symlink issues) 
+then configure and make:
+
+
+cd /c/deps/
+tar xvfz openssl-1.0.1k.tar.gz
+cd openssl-1.0.1k
+./Configure no-zlib no-shared no-dso no-krb5 no-camellia no-capieng no-cast no-cms no-dtls1 no-gost no-gmp no-heartbeats no-idea no-jpake no-md2 no-mdc2 no-rc5 no-rdrand no-rfc3779 no-rsax no-sctp no-seed no-sha0 no-static_engine no-whirlpool no-rc2 no-rc4 no-ssl2 no-ssl3 mingw
+make
+
+
+
+2.2 Berkeley DB: http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz
+We'll use version 4.8 to preserve binary wallet compatibility.
+From a MinGW shell unpack the source archive, configure and make:
+
+
+cd /c/deps/
+tar xvfz db-4.8.30.NC.tar.gz
+cd db-4.8.30.NC/build_unix
+../dist/configure --enable-mingw --enable-cxx --disable-shared --disable-replication
+make
+
+
+
+2.3 Boost: http://sourceforge.net/projects/boost/files/boost/1.61.0/
+Download either the zip or the 7z archive, unpack boost inside your C:\deps folder, then bootstrap and compile 
+from a Windows command prompt:
+
+
+cd C:\deps\boost_1_61_0\
+bootstrap.bat mingw
```

Comment:
`bootstrap.bat mingw` doesn't work.  Fails with an unknown toolset error (displayed in bootstrap.log file.

if I use `bootstrap.bat gcc` it works.